server/routes/temporaryabsences/checkTemporaryAbsenceController.test.ts

```diff
@@ -36,7 +40,37 @@ describe('GET checkTemporaryAbsence', () => {
     return request(app).get('/prisoners/A1234AA/check-temporary-absence').expect(302).expect('Location', '/autherror')
   })
 
+  it('should only display back navigation of the back link type', () => {
+    config.showBreadCrumb = false
+    createAppWithAllRoutes()
```

Comment:
this is a bit gnarly - the var on app is a bit of a smell that suggests you have some unnecessary coupling - you need to call a method with a side effect or an apparently unrelated variable (`app`) will be invalid. 
 
Would suggest getting rid of `app` variable and returning it from  `createAppWithAllRoutes()` passing and moving breadcrumb setup into there: 

so you can just do something like:

`return request(createApp({breadcrumb: true}))`

where create app looks like: 
```
const createApp = ({ breadcrumb}: {breadcrumb: boolean}) => {
   config.showBreadCrumb = breadcrumb
  return appWithAllRoutes({
    services: { temporaryAbsencesService, raiseAnalyticsEvent },
    roles: [Role.PRISON_RECEPTION],
  })
}

```

Reply:
I did think of doing that at first but then felt it was more readable having a separate variable.